jaxb-ri/codemodel/codemodel/pom.xml

```diff
@@ -26,46 +26,22 @@
 
     <properties>
         <findbugs.exclude>${project.basedir}/exclude-codemodel.xml</findbugs.exclude>
+        <base.java.level>7</base.java.level>
+        <upper.java.level>9</upper.java.level>
     </properties>
 
     <dependencies>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
+            <version>4.12</version>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>com.google.code.javaparser</groupId>
             <artifactId>javaparser</artifactId>
+            <version>1.0.11</version>
```

Comment:
this dependency is being used only here, so should be removed from former (jaxb) parent